synfig-core/src/tool/renderprogress.cpp

```diff
@@ -69,15 +72,18 @@ bool RenderProgress::amount_complete(int current_frame, int frames_count)
         }
         last_timepoint_ = Clock::now();
 
+		int total_frames_count = total_repeats * frames_count;
+		int r_current_frame = current_frame + frames_count * repeated_;
+
         int percentage_completed = 100;
-        if (frames_count > 0)
+        if (total_frames_count > 0)
         {
-            percentage_completed = 100 * current_frame / frames_count;
+            percentage_completed = (100 * r_current_frame) / total_frames_count;
```

Comment:
I am not sure if this should be percentage completed for the current render or for the whole thing.

Reply:
I think the whole think is preferable.
Can you check remaining time calculation too?

```
./solid.sifz ==> ./solid.null: Iteration 5374: Frame 1 of 1 (53%). Remaining time: 0s 
```

Reply:
Sure, I will fix the remaining time to display the total remaining time as well. I made the changes, will commit after testing.